package.json

```diff
@@ -48,6 +48,10 @@
   "jest": {
     "collectCoverageFrom": [
       "lib/**/*.js"
+    ],
+    "testPathIgnorePatterns": [
```

Comment:
if you don't add these lines `npm test` can run benchmark as well? I think that'd be also good.

Reply:
Yes but it will run as jest tests and i thought that we just want to add screenshots of results to `README.md`

Reply:
I don't mind that either. But I'm fine either way.